README.md

```diff
@@ -24,9 +24,7 @@ components that conform to Marshmallow's Design System - "_S'mores_".
 ## Install
 
 ```bash
-$ yarn add @mrshmllw/smores-react
```

Comment:
moving the project back to `npm` wouldn't prevent people from installing it using `yarn`

Reply:
shall we leave the **README.md** as it is?

Reply:
That makes sense, although we should punish them.